tests/app/dao/test_services_dao.py

```diff
@@ -1099,3 +1071,79 @@ def create_email_sms_letter_template():
     template_two = create_template(service=service, template_name='2', template_type='sms')
     template_three = create_template(service=service, template_name='3', template_type='letter')
     return template_one, template_three, template_two
+
+
+@freeze_time("2019-12-02 12:00:00.000000")
```

Comment:
Do we need to freeze_time for this test?

Reply:
I think we do, to show that most notifications are in good date range, while notifications for service_6 are just a bit too old and do not make it to results.